### DISCUSSION TITLE
Is it possible to customize new operators for my project?

### DISCUSSION THREAD

**Discussion by AndresPalacio** [react-querybuilder/react-querybuilder#471] [Q&A]
Hi, currently I haven't found a way to implement a new operator for my case isPrime (example) to generate a SpEL. Is there a way to configure this?

**Comment by jakeboone02** [comment]
Are you wanting to generate an expression with standard SpEL operators or a custom function? For example would `isPrime` be directly translatable to SpEL?

**Reply by AndresPalacio** [reply]
I would just like to know the possibility of being able to customize operators. The tool is already powerful enough for my requirements with the help of aggregations, but for the isPrime example, it could be something like the following, taking this Java logic to React.
Is a member is an example of how isprime is built 

```

public class Society {

    private String name;

    public static String Advisors = "advisors";
    public static String President = "president";

    private List<Inventor> members = new ArrayList<Inventor>();
    private Map officers = new HashMap();

    public List getMembers() {
        return members;
    }

    public Map getOfficers() {
        return officers;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isMember(String name) {
        for (Inventor inventor : members) {
            if (inventor.getName().equals(name)) {
                return true;
            }
        }
        return false;
    }
```

**Reply by jakeboone02** [reply]
You can use custom operators in RQB (see https://react-querybuilder.js.org/docs/tips/managing-operators), but I'm not sure that's what you're talking about.

To generate SpEL expressions that use custom functions like that you'd have to use a custom [`ruleProcessor`](https://react-querybuilder.js.org/docs/utils/export#rule-processor). But honestly I can't tell from your question if that's what you mean either.

**Comment by AndresPalacio** [comment]
In addition to this, could you provide me with examples of the use of various aggregations in a project?

**Reply by jakeboone02** [reply]
RQB doesn't really handle aggregations, just conditions/filters like in a SQL `WHERE` clause.
